core/tee/tee_svc_cryp.c

```diff
@@ -1664,27 +1673,29 @@ static TEE_Result copy_in_attrs(struct user_ta_ctx *utc,
 	TEE_Result res = TEE_SUCCESS;
 	size_t size = 0;
 	uint32_t n = 0;
+	struct utee_attribute *usr_attrs_bbuf = NULL;
 
 	if (MUL_OVERFLOW(sizeof(struct utee_attribute), attr_count, &size))
 		return TEE_ERROR_OVERFLOW;
 
 	usr_attrs = memtag_strip_tag_const(usr_attrs);
```

Comment:
remove this line to preserve `usr_attrs` tag. `copy_from_user()` will take care of it.

Reply:
oh yes, I will remove this